deploy/rbac.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
This is the combination of service_account, role, and role_binding.yaml files -- no changes